src/main/groovy/com/blackbaud/templates/tasks/RestProject.groovy

```diff
@@ -153,7 +162,23 @@ authorization.filter.enable=false
     }
 
     void createCrudResource(String resourceName, boolean addEntity, boolean addWireSpec) {
-        addResourceAndSupportingClasses(resourceName, "crud", addWireSpec)
+        addResourceAndSupportingClasses(resourceName, addWireSpec)
+
+        File resourceFile = basicProject.findFile("${resourceName}Resource.java")
+        FileUtils.appendAfterLine(resourceFile, "class", """\
```

Comment:
just curious, does this just search for the first line with 'class' in it?

Reply:
yep